docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -35,9 +35,7 @@ Software
 Access
 ~~~~~~
 
-- A Read the Docs account (`registration <https://readthedocs.org/accounts/signup/>`_),
-  with `maintainer access <https://readthedocs.org/dashboard/flocker/users/>`_ to the Flocker project.
-- Access to `Google Cloud Storage`_ using `gsutil`_.
+- Access to `Google Cloud Storage`_ and `Amazon S3` using `gsutil`_.
```

Comment:
Does gsutil use the same config files for authentication as the other s3 tools we looked at or does it have its own?